### PR TITLE
Added comment about running `run` or `package` for resources

### PR DESCRIPTION
**Pull request by reactormonk**
Should solve https://github.com/sbt/sbt/issues/1832

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @reactormonk,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: src/reference/04-Howto/03-Howto-Generating-Files.md [hunk 1]
```diff
@@ -103,6 +103,11 @@ resourceGenerators in Compile += Def.task {
 }.taskValue
 ```
 
+Executing `run` (or `package`) will add a file `demo` to `resourceManaged`,
+which is `target/scala-*/resource_managed"`. By default, generated resources are
+not included in the packaged source artifact. To do so, add them as you would
+other mappings. See [Adding files to a package][modify-package-contents].
```
Comment (jvican) [inline src/reference/04-Howto/03-Howto-Generating-Files.md]: Can we add a small line here saying: "Note that running `compile` is not sufficient for `resourceGenerators` to run."

Comment (reactormonk) [inline src/reference/04-Howto/03-Howto-Generating-Files.md]: I figured that would be a bit too much text, and this would be sufficient.

**Comment by dwijnand** [comment]
Thanks, @reactormonk.

**Pull request merged**